packages/@react-spectrum/color/docs/ColorArea.mdx

```diff
@@ -0,0 +1,310 @@
+import {Label} from '@react-spectrum/label';
+import {View} from '@react-spectrum/view';
+```
+
+---
+category: Color
+keywords: [color area]
+---
+
+# ColorArea
+
+<PageDescription>{docs.exports.ColorArea.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorArea']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/color-area/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ColorArea defaultValue="#7f0000" />
+```
+
+## Value
+
+A ColorArea requires either an uncontrolled default value or a controlled value, provided using the `defaultValue` or `value` props respectively.
+The value provided to either of these props should be a color string or <TypeLink links={colorTypes.links} type={colorTypes.exports.Color} /> object.
+`xChannel` and `yChannel` props may also be provided to specify which color channels the color area should display, and the direction of each channel in the color gradient.
+This must be one of the channels included in the color value, for example, for RGB colors, the "red", "green", and "blue" channels are available.
+For a full list of supported channels, see the [Props](#props) table below.
+
+In the example below, the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.parseColor} /> function is used to parse the initial color from a HSL string
+so that `value`'s type remains consistent.
+
+```tsx example
+import {parseColor} from '@react-stately/color';
+
+function Example() {
+  let [value, setValue] = React.useState(parseColor('hsl(0, 100%, 50%)'));
+  return (
+    <Flex gap="size-300" wrap>
+      <div>
+        <Label id="hsl-uncontrolled-id">x: Saturation, y: Lightness (uncontrolled)</Label>
+        <ColorArea
+          aria-labelledby="hsl-uncontrolled-id"
+          defaultValue={value}
+          xChannel="saturation"
+          yChannel="lightness" />
+      </div>
+      <div>
+        <Label id="hsl-controlled-id">x: Saturation, y: Lightness (controlled)</Label>
+        <ColorArea
+          aria-labelledby="hsl-controlled-id"
+          value={value}
+          onChange={setValue}
+          xChannel="saturation"
+          yChannel="lightness" />
+      </div>
+    </Flex>
+  );
+}
+```
+
+## Labeling
+
+By default, ColorArea uses an `aria-label` for the localized string "Color Picker",
+which labels the visually hidden `<input>` elements for the two color channels, or on mobile devices,
+the group containing them. If you wish to override this with a more specific label, an `aria-label` or
+`aria-labelledby` prop may be passed to further identify the element to assistive technologies.
+For example, for a ColorArea that adjusts a background color you might pass the `aria-label` prop,
+"Background color", which ColorArea will return as the `aria-label` prop "Background color, Color picker"
+for each of the `<input>` elements, or on mobile devices, the group containing them. If you provide your own
```

Comment:
Similar to the feedback in the useColorArea docs, IMO some extraneous language here since you've mentioned it above already:
```suggestion
"Background color", which ColorArea will return as the `aria-label` prop "Background color, Color picker". If you provide your own
```